daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -542,7 +563,48 @@ func (devices *DeviceSet) migrateOldMetaData() error {
 	return nil
 }
 
+// Cleanup deleted devices during startup. It assumes that all the
```

Comment:
Right. This comment can be changed now. Initially I was doing cleanup only during startup and that's what this comment reflects. Later I also introduced a thread to do the cleanup. I will update this comment.